### PR TITLE
adding libraries

### PR DESCRIPTION
**Pull request by nickjacobson**
These 2 libraries seem to support Python 3:

1. https://pypi.python.org/pypi/diamond - The download file is called diamond-4.0.451-py2.py3-none-any.whl
2. https://pypi.python.org/pypi/filechunkio - The pypi page says "Python 2.6+ is required" which would seem to include Python 3. And there's a checkin here that appears to have added support for Python 3: https://bitbucket.org/fabian/filechunkio/pull-requests/1/python-3-support/diff#comment-None

### CODE REVIEW

File: caniusepython3/overrides.json [hunk 1]
```diff
@@ -3,10 +3,12 @@
     "bcdoc": "https://github.com/boto/bcdoc/blob/master/tox.ini",
     "certifi": "https://pypi.python.org/pypi/certifi",
     "collective.recipe.template": "https://pypi.python.org/pypi/collective.recipe.template",
+    "diamond": "https://pypi.python.org/pypi/diamond",
     "django-social-auth": "https://pypi.python.org/pypi/python-social-auth",
     "dnspython": "https://pypi.python.org/pypi/dnspython3",
     "extras": "https://pypi.python.org/pypi/extras",
     "fabric": "https://pypi.python.org/pypi/Fabric3",
+    "filechunkio": "https://pypi.python.org/pypi/filechunkio",
```
Comment (brettcannon) [inline caniusepython3/overrides.json]: Please update this to have the value point to the commit that shows Python 3 support.

**Comment by brettcannon** [comment]
The PR LGTM overall, it just needs a tweak to the link pointing to the evidence of one of the packages.

**Comment by nickjacobson** [comment]
Hmm, unfortunately I don't know what URL would point it to a commit.

**Comment by brettcannon** [comment]
https://bitbucket.org/fabian/filechunkio/commits/5a1d0b614e506e5198090a5201770e2524fa8720

**Comment by brettcannon** [comment]
Thanks!

**Pull request merged**